libs/velo-external-db-core/src/service/data.spec.ts

```diff
@@ -12,7 +12,7 @@ describe('Data Service', () => {
         driver.givenListResult(ctx.entities, ctx.collectionName, ctx.filter, ctx.sort, ctx.skip, ctx.limit, ctx.defaultProjection)
         driver.givenCountResult(ctx.total, ctx.collectionName, ctx.filter)
         
-        return expect(env.dataService.find(ctx.collectionName, ctx.filter, ctx.sort, ctx.skip, ctx.limit, ctx.defaultProjection)).resolves.toEqual({
+        return expect(env.dataService.find(ctx.collectionName, ctx.filter, ctx.sort, ctx.skip, ctx.limit, ctx.defaultProjection, true)).resolves.toEqual({
```

Comment:
Can you add a test here to `returnTotalCount: false`?

Reply:
si